### DISCUSSION TITLE
Unphysical horizontal velocity pattern in vicinity of vegetation

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#13027] [Q&A] [answered]
Dear FDS developers,

The horizontal velocity profile of an MO wind profile blowing towards a tree gives the following results:

t = 0:
![before](https://github.com/firemodels/fds/assets/137812007/dc49021a-7c61-47d8-ba56-c7350058b116)
t > 0:
![after](https://github.com/firemodels/fds/assets/137812007/d5f9e45e-caf4-48a0-819c-bbe232467add)
![close](https://github.com/firemodels/fds/assets/137812007/9e9419d7-a295-495d-9044-a288596d97e6)

The general behaviour is, I believe, correct. However, in the vicinity of the treetop, we observe an unphysical pattern. This is not due to Smokeview, as the `CELL_CENTERED` values also show such patterns. 

Based on older discussions, it seems that this may be due to discretization errors. However, I find the magnitude difference of these patterns at the treetop unusually high. Is there any way to reduce this?

The Lagrangian particles that model the tree are given a constant drag coefficient of 2.8. The finest meshes that contain the tree are $1.2 \times 1.2 \times 2.4 \ \mathrm{m}^3$ and have grid cells of $7.5 \times 7.5 \times 7.5 \ \mathrm{cm}^3$. The tree has $D = 1.19 \ \mathrm{m}$, $H = 3.96 \ \mathrm{m}$.

**Comment by rmcdermo** [comment, marked as answer]
This is simply numerical (dispersion error) from the central differencing scheme in the U-component momentum equation.  This is inevitable with low-order central schemes at practical grid resolutions.  Look at either the velocity magnitude or kinetic energy, these should look better.

**Reply by r-broek** [reply]
Thank you!
